src/sickle.ts

```diff
@@ -263,6 +266,29 @@ class Annotator {
     this.emit(': void;\n');
   }
 
+  private visitEnum(node: ts.EnumDeclaration) {
+    this.emit('/** @typedef {number} */\n');
```

Comment:
Mh. Seems not worth the trouble, Closure doesn't like assignment into the enum later on.